core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -473,6 +474,7 @@ Builder populate() {
       defineMethod(EXP, "exp", NullPolicy.STRICT);
       defineMethod(POWER, "power", NullPolicy.STRICT);
       defineMethod(LN, "ln", NullPolicy.STRICT);
+      defineMethod(LOG, "log", NullPolicy.STRICT);
```

Comment:
Would this be simpler if done with an implementor that translates 'log(x, y)' to 'ln(x) / ln(y)'? Constant reduction could kick in, we could potentially optimize for bases such as e, 2, 10, and we would not need to add to `SqlFunctions`.

Method implementor is kind of a hammer.